pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [tool.poetry]
     name = "check-python-versions"
-    version = "1.0.0"
```

Comment:
It's not the version that's a problem, it's that this project (check-python-versions) doesn't use Poetry and therefore it shouldn't have Poetry configuration in its top-level pyproject.toml.

Please move this file into `tests/sources/data/` or a similarly named subdirectory if it's needed by the test suite.

Reply:
it's clear that this project doesn't use poetry to build it.

I've made the file to do a live check with check-python-versions.py , so it could be related to test_cli.py , I've to look at it.